### ISSUE TITLE
[Request] Virtual Rumble

### ISSUE THREAD

**Issue by tony971** [libretro/RetroArch#4328]
The idea is to shake the screen or play a rumble sound whenever a controller is supposed to rumble. This will allow using rumble features while using hardware that does not support rumble.

**Comment by meepingsnesroms** [comment]
First android and ios rumble should be implemented because they actually can rumble.

**Comment by tony971** [comment]
That doesn't make this invalid.

**Comment by geekley** [comment]
Yes, please! This would provide accessibility to people who don't have controllers with vibration support. And it's not just for "cosmetic" purposes -- some games like OoT lock you out of certain features when you don't have rumble.

Shaking screen with sound effect would be ideal, but any kind of visual or audio rumble would be fine -- even just an icon or blinking screen border overlay or whatever is easier to implement -- as long as it works in fullscreen mode, unlike the similar feature of N-rage plugin for PJ64.
